app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -0,0 +1,243 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:cocoon_service/src/request_handling/exceptions.dart';
+import 'package:meta/meta.dart';
+
+import '../../cocoon_service.dart';
+import '../model/appengine/service_account_info.dart';
+import '../model/luci/buildbucket.dart';
+import 'buildbucket.dart';
+
+/// Class with utilities to interact with LUCI buildbucket to get, trigger
```

Comment:
Nit: remove "with utilities" since a service class is expected to be able to abstract the interactions with an external dependency.

Reply:
Done